neo/test/iotest/test_nixio.py

```diff
@@ -195,7 +199,7 @@ def compare_signals_das(self, neosignals, data_arrays):
             for da in data_arrays:
                 if da.metadata.name == nixname:
                     dalist.append(da)
-            _, nsig = np.shape(sig)
+                nsig = np.shape(sig)[-1]
```

Comment:
I think this is indented one level too deep. Would make more sense outside of the foor loop, no?

Reply:
Yes, good catch.